### PR TITLE
Task/hw 60987 expose clear instance

### PR DESCRIPTION
**Pull request by manikandan-altmtrk**
clear instance implemented.

### CODE REVIEW

File: receiptui/src/main/java/com/hyperwallet/android/ui/receipt/HyperwalletReceiptUi.java [hunk 1]
```diff
@@ -65,4 +65,7 @@ public Intent getIntentListPrepaidCardReceiptActivity(@NonNull final Context con
         return intent;
     }
 
+    public static void clearInstance() {
+        sInstance = null;
```
Comment (peter-joseph) [inline receiptui/src/main/java/com/hyperwallet/android/ui/receipt/HyperwalletReceiptUi.java]: Hyperwallet.clearInstance();

File: transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java [hunk 2]
```diff
@@ -102,4 +102,8 @@ public Intent getIntentAddTransferMethodActivity(@NonNull final Context context,
         intent.putExtra(EXTRA_TRANSFER_METHOD_PROFILE_TYPE, profileType);
         return intent;
     }
+
+    public static void clearInstance() {
+        sInstance = null;
```
Comment (peter-joseph) [inline transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java]: Hyperwallet.clearInstance();

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/HyperwalletTransferUi.java [hunk 3]
```diff
@@ -52,13 +52,16 @@ public static synchronized HyperwalletTransferUi getInstance(
...
     }
 
     public Intent getIntentCreateTransfer(@NonNull final Context context, @NonNull final String sourceToken) {
         Intent intent = new Intent(context, CreateTransferActivity.class);
         intent.putExtra(CreateTransferActivity.EXTRA_TRANSFER_SOURCE_TOKEN, sourceToken);
         return intent;
     }
+
+    public static void clearInstance() {
+        sInstance = null;
```
Comment (peter-joseph) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/HyperwalletTransferUi.java]: Hyperwallet.clearInstance();

File: transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java [hunk 4]
```diff
@@ -102,4 +103,9 @@ public Intent getIntentAddTransferMethodActivity(@NonNull final Context context,
...
+        Hyperwallet.clearInstance();
+    }
```
Comment (jsaini-hw) [inline transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java]: Add clearInstance method inside HyperwalletInsight class and add HyperwalletInsight.clearInstance() here.

Comment (manikandan-altmtrk) [reply transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/HyperwalletTransferMethodUi.java]: It is merged. May I do in another PR?

**Pull request merged**